themes/SuiteR/css/style.css

```diff
@@ -5173,6 +5173,9 @@ div.target-list-table {
     padding: 10px;
 }
 
+table tr#delegates_search {
```

Comment:
All sorted out :)

________________________________
From: Matt Lorimer <notifications@github.com>
Sent: 06 July 2016 11:57:12
To: salesagility/SuiteCRM
Cc: Jerzy Brzezinski; Author
Subject: Re: [salesagility/SuiteCRM] Adding Events Delegates Filter via Subpanel (#1709)


In themes/SuiteR/css/style.css<https://github.com/salesagility/SuiteCRM/pull/1709#discussion_r69710639>:

> @@ -5173,6 +5173,9 @@ div.target-list-table {
>      padding: 10px;
>  }
>
> +table tr#delegates_search {


is this only relevant to the current theme or any theme?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/salesagility/SuiteCRM/pull/1709/files/d5745dd3d52737bf7a37996b4757ba21b6cd7bd2#r69710639>, or mute the thread<https://github.com/notifications/unsubscribe/AKP8DDfbTlZ4vEf4GFW2_E78KscxNGA6ks5qS4oIgaJpZM4JF9xd>.